### PR TITLE
change path of the pdf

### PR DESCRIPTION
**Pull request by YouQuan2007**

### CODE REVIEW

File: src/sample/classes/Student.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) throws FileNotFoundException{
         list1.add("Kotlin");
         list1.add("Android");
         
-        String path = "Desktop/CourseList.pdf"; 
+        String path = "C:/Users/User/Downloads/CourseList.pdf";
```
Comment (Michelle-Lohwt) [inline src/sample/classes/Student.java:19]: I don't think everyone has `users/user` path, is it possible to allow users to choose where to download?

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
I don't think everyone has users/user path, is it possible to allow users to choose where to download?

File: src/sample/classes/Student.java (hunk 1 above)
Comment (YouQuan2007) [reply src/sample/classes/Student.java:19]: i am not sure about this, if let user choose where to download means will involve php dy isnt it? maybe we can discuss a way forward after 232 class

Comment (Michelle-Lohwt) [reply src/sample/classes/Student.java:19]: Are you going to change this @YouQuan2007 ?

Comment (YouQuan2007) [reply src/sample/classes/Student.java:19]: @ziqiang2344 have solve this issue dy, so i guess this pull request can delete

**Pull request closed without merging**